tests/Microsoft.AspNet.SignalR.Client.Portable.Tests/packages.config

```diff
@@ -1,8 +1,41 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="Microsoft.NETCore.Platforms" version="1.0.1" targetFramework="net45" />

```

Comment:
Not sure. At the moment you can't open this project with VS2017 due to store and WinRT projects and I don't know if VS2015 supports PackageReference.